python/ql/test/query-tests/Security/CWE-327/InsecureProtocol.py

```diff
@@ -0,0 +1,35 @@
+import ssl
+from pyOpenSSL import SSL
```

Comment:
Can you add tests for `from ssl import PROTOCOL_SSLv2` and similar.

Reply:
Done (at bottom of file).